packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -71,8 +73,7 @@ export const processConsolidatedDataWebhook = async ({
     };
 
     // send it to the customer and update the WH request status
-
-    if (!isWebhookDisabled(currentPatient.data.cxConsolidatedRequestMetadata)) {
+    if (!isWebhookDisabled(currentPatient.data.cxConsolidatedRequestMetadata) && shouldSendWh) {
```

Comment:
would probably be good to also add the 
`// TODO 858 indicate this was not really sent to the customer` on the else case here. Much like we do in the other place where a webhook isn't actually sent